conga-aem-definitions/src/main/templates/aem-cms/aem-cms-publish-replicationagents.json.hbs

```diff
@@ -28,6 +28,7 @@
         "triggerReceive": "true",
         "triggerSpecific": "true",
         "noVersioning": "true",
+        "aliasUpdate": {{defaultIfEmpty this.aliasUpdate false}},
```

Comment:
please add a documentation for this new parameter to
https://github.com/wcm-io-devops/conga-aem-definitions/blob/43210df17d3639ebd2b7879d5a38f27f89ced4f3/conga-aem-definitions/src/main/roles/aem-cms.yaml#L197-L199

and it's also better to define the default value there, and insert the value here directly without using "defaultIfEmpty" - this is only required when there is no other way to define a default value.

Reply:
done